### PR TITLE
Add compat data to input-week

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/week

### CODE REVIEW

File: html/elements/input/week.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "This feature is not implemented yet. See <a href='https://bugzil.la/888320'>bug 888320</a> and <a href='https://wiki.mozilla.org/TPE_DOM/Date_time_input_types'>TPE DOM/Date time input types</a>."
```
Comment (ddbeck) [inline html/elements/input/week.json]: This is `false` for both desktop and Android Firefoxes and @wbamberg's [comment]([https://github.com/mdn/browser-compat-data/pull/2200#discussion_r197279506]) on #2200 applies to both as well

File: html/elements/input/week.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-week": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/week",
```
Comment (ddbeck) [inline html/elements/input/week.json]: Needs `"description": "<code>type=\"week\"</code>"`

**Comment by maboa** [comment]
This should be sorted now! Thanks :)

File: html/elements/input/week.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "This feature is not implemented yet. See <a href='https://bugzil.la/888320'>bug 888320</a>."
```
Comment (ddbeck) [inline html/elements/input/week.json]: One last nit, sorry. Can we change this note to only "See <a href='https://bugzil.la/888320'>bug 888320</a>." Thank you, @maboa!

**Pull request merged**